python/gtsam/tests/test_Sim2.py

```diff
@@ -46,7 +46,7 @@ def test_align_poses_along_straight_line(self) -> None:
 
         wToi_list = [wTo0, wTo1, wTo2]
 
-        we_pairs = Pose2Pairs(list(zip(wToi_list, eToi_list)))
+        we_pairs = list(zip(wToi_list, eToi_list))
```

Comment:
does pair translate to list? or tuple? or either?

Reply:
Pair should now be a tuple.